frontend/packages/integration-tests-cypress/reporter-config.json

```diff
@@ -1,7 +1,14 @@
 {
-  "reporterEnabled": "spec, mocha-junit-reporter",
+  "reporterEnabled": "mocha-junit-reporter, mochawesome",
   "mochaJunitReporterReporterOptions": {
     "mochaFile": "../../gui_test_screenshots/junit_cypress-[hash].xml",
     "toConsole": false
+  },
+  "mochawesomeReporterOptions": {
+    "reportDir": "../../gui_test_screenshots/",
+    "reportFilename": "cypress_report",
```

Comment:
nit: `cypress-report` so it follows the same format as the protractor report name

Reply:
note: these are for the temp .json files which are deleted at the end.  The 'cypress-report' is passed in cmd line from package.json :-)